.github/workflows/build_all.yml

```diff
@@ -373,193 +373,6 @@ jobs:
-
-      - uses: ./.github/workflows/composite/docker-builder-agw
-        id: agw-build-docker
-        with:
-          REGISTRY_USERNAME: ${{ secrets.JFROG_USERNAME }}
-          REGISTRY_PASSWORD: ${{ secrets.JFROG_PASSWORD }}
-          REGISTRY: ${{ env.DOCKER_REGISTRY }}
-          IMAGE: ${{ env.DOCKER_IMAGE }}
-          DOCKERFILE: ${{ env.DOCKER_FILE }}
-          PUSH: ${{ github.event_name == 'push' }}
-
-  agw-container-build-go:
-    if: github.repository_owner == 'magma'
-    runs-on: ubuntu-latest
-    env:
-      DOCKER_REGISTRY: agw-ci.artifactory.magmacore.org
-      DOCKER_IMAGE: gateway_go
-      DOCKER_FILE: feg/gateway/docker/go/Dockerfile
-    outputs:
-      digest: ${{ steps.agw-build-docker.outputs.digest }}
-    steps:
-      - run: echo "DOCKER_REGISTRY=agw-test.artifactory.magmacore.org" >> $GITHUB_ENV
-        if: github.ref == 'refs/heads/master'
-
-      - run: echo "Publishing images to ${{ env.DOCKER_REGISTRY }}"
-
-      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
-
-      - uses: ./.github/workflows/composite/docker-builder-agw
-        id: agw-build-docker
-        with:
-          REGISTRY_USERNAME: ${{ secrets.JFROG_USERNAME }}
-          REGISTRY_PASSWORD: ${{ secrets.JFROG_PASSWORD }}
-          REGISTRY: ${{ env.DOCKER_REGISTRY }}
-          IMAGE: ${{ env.DOCKER_IMAGE }}
-          DOCKERFILE: ${{ env.DOCKER_FILE }}
-          PUSH: ${{ github.event_name == 'push' }}
-
-  agw-container-build-is-successful:
-    runs-on: ubuntu-latest
-    if: success()
-    needs:
-      - agw-container-build-c
-      - agw-container-build-python
-      - agw-container-build-go
-    outputs:
-      digest-c: ${{ needs.agw-container-build-c.outputs.digest }}
-      digest-python: ${{ needs.agw-container-build-python.outputs.digest }}
-      digest-go: ${{ needs.agw-container-build-go.outputs.digest }}
-    steps:
-      - run: echo "Successfully built AGW containers."
-
-  agw-container-build:
-    name: agw container build
-    runs-on: ubuntu-latest
-    if: always() && github.repository_owner == 'magma'
-    needs:
-      - agw-container-build-is-successful
-    steps:
-      - name: Check completion of all build steps
-        run: |
-          if [ "${{ needs.agw-container-build-is-successful.result }}" = "success" ];
-          then
-            exit 0
-          else
-            echo "Exit status of some previous job(s) was ${{ needs.agw-container-build-is-successful.result }}"
-            exit 1
-          fi
-
-      - name: Show docker image digests
-        run: |
-          echo "digest-c:      ${{ needs.agw-container-build-is-successful.outputs.digest-c }}"
-          echo "digest-python: ${{ needs.agw-container-build-is-successful.outputs.digest-python }}"
-          echo "digest-go:     ${{ needs.agw-container-build-is-successful.outputs.digest-go }}"
-
-      - name: Trigger integration test workflow
```

Comment:
How are the integ tests triggered now? The description mentions `workflow_run`, but I don't find that currently.

Reply:
The PR for the receiving side https://github.com/magma/magma/pull/14353 also needs adoptions. :construction: However, those are not yet implemented. I would update that PR (#14353) once this one is final to avoid doing changes to it over and over again.